### PR TITLE
WIP: Switch to CRI v1 protocol

### PR DESCRIPTION
**Pull request by saschagrunert**
#### What type of PR is this?

/kind api-change

#### What this PR does / why we need it:
We now use the graduated CRI v1 protocol instead of the v1alpha2.

#### Which issue(s) this PR fixes:

None
#### Special notes for your reviewer:
None
#### Does this PR introduce a user-facing change?

```release-note
Changed Container Runtime Interface implementation to only support `v1` instead of `v1alpha2`.
```

### CODE REVIEW

File: v1/lib/rename.go [hunk 1]
```diff
@@ -0,0 +1,116 @@
+package lib
```
Comment (haircommander) [inline v1/lib/rename.go:1]: why'd we drop `internal`? doesn't that mean other libraries can vendor `v1/lib` which could mess with our versioning?

Comment (saschagrunert) [reply v1/lib/rename.go:1]: Its impossible to import an `internal` package from a sub-package of the project from another part of the project.

**Comment by saschagrunert** [comment]
Will open a new one

**Pull request closed without merging**